### ISSUE TITLE
Allow setting of HTML Attribute and CSS features at the <div class="page"> level

### ISSUE THREAD

**Issue by dbolack-ab** [naturalcrit/homebrewery#3901]
### Your idea:

Presently, when a user wants to change CSS parameters of a Page classed div, it has to be done via `page :has()` mechanisms. While this works pretty well, it's not the most intuitive thing for folks new to CSS. 
Its primary failing is it only permits setting CSS Properties to predefined values.

This issue proposes that we extend the single-branch mustache syntax to be appendable to a `\page` and inserted into the page classed div's style, class, and arbitrary add-on attributes as desired.

Additionally, to support setting values on the first page, we need to consider dropping a completely empty Page 1, treating the `\page` definition as the start of the page, not the end. By only dropping the empty page body, we maintain backward compatibility.

example syntax:

```
\page{--TOC:"'exclude'",wide,XCUSTOMATTR="'Foo'"}
```

* I may have the quoting syntax inverted in the example.

**Comment by 5e-Cleric** [comment]
I agree this would make things easier for users, if they find out about this feature.

**Comment by 5e-Cleric** [comment]
This was implemented
